src/NLog/Targets/EventLogTarget.cs

```diff
@@ -437,11 +448,10 @@ private void CreateEventSourceIfNeeded(string fixedSource, bool alwaysThrowError
             catch (Exception exception)
             {
                 InternalLogger.Error(exception, "Error when connecting to EventLog.");
-                if (alwaysThrowError || exception.MustBeRethrown())
+                if (alwaysThrowError || LogManager.ThrowExceptions)
```

Comment:
this is a bugfix?

Reply:
Not public

Reply:
also this one, copy code or it should be public

Reply:
Don't like to pollute with public extension methods on non-NLog-objects. Have now introduced a new public method `LogManager.MustBeRethrown(Exception)`